core/infra/config/peer_service_with_json.cpp

```diff
@@ -124,11 +124,14 @@ size_t PeerServiceConfig::getMaxFaulty() {
 bool PeerServiceConfig::isMyActive() {
     return is_active;
 }
+
 bool PeerServiceConfig::active() {
-    is_active == true;
+    return is_active == true;
 }
+
 bool PeerServiceConfig::stop(){
     is_active = false;
+    return active();
```

Comment:
`return active()` isn't needed by this fucntion.
It is only `is_active = false`